examples/directml/llm/user_script.py

```diff
@@ -27,6 +28,25 @@ def __getitem__(self, idx):
 # DECODER
 # -----------------------------------------------------------------------------
 
+def find_weight_by_subname(v_dict, subname):
+    """
+    Find weight in dictionary by subname.
+
+    Arguments:
+        v_dict -- dictionary to search.
+        subname -- subname to search for.
+
+    Raises:
+        ValueError: subname not found, or duplicates.
+
+    Returns:
+        The weights for an element with the given subname.
+    """
```

Comment:
Those comments are not relevant for this code snippet since it's not a function anymore. we can remove them